rollup.config.js

```diff
@@ -24,7 +24,6 @@ export default {
   plugins: [
     terser({
       warnings: true,
-      module: true,
```

Comment:
Why this change?

Reply:
Because it's the default now: [rollup-plugin-terser v4.0.0](https://github.com/TrySound/rollup-plugin-terser/releases/tag/v4.0.0)

>automatically set module: true for the esm format ([Tiger Oakes](https://github.com/NotWoods) in [#22](https://github.com/TrySound/rollup-plugin-terser/pull/22))